ldappool/__init__.py

```
@@ -0,0 +1,504 @@
+                con.inUse = False
+                self.put(con)
+                self.__revamp__()
+                raise
+        return con
+
+    def put(self, connection):
+        if not self._lock.acquire(timeout=1):
+            raise LDAPLockTimeout("Could not acquire pool lock")
+        if connection.inUse:
+            connection.giveback()
+        if not connection in self._pool:
```

Ruff (0.14.10)

Test for membership should be `not in`

Convert to `not in`

(E713)